content/tracing/setup/docker.md

```diff
@@ -46,35 +46,85 @@ As with DogStatsD, traces can be submitted to the agent from other containers ei
 ### Using Docker links
```

Comment:
In the sentence above we list "host IP" before "docker links" - let's either change the sentence or switch the order of these.

Reply:
Updated 👍